src/index.js

```diff
@@ -0,0 +1,84 @@
+const crypto = require('crypto');
+const assert = require('assert');
+
+const key = process.env.HASHING_KEY;
+assert(key, 'HASHING_KEY not provided');
+
+const ACTION = {
+  ENCRYPT: 'ENCRYPT',
+  DECRYPT: 'DECRYPT'
+};
+
+/**
+   * Encrypt/Decrypt incoming data depending on action
+   * @param  {Array|string|boolean|object} data - data to encrypt/decrypt.
+   * In case of encryption, data can be Array of string or string.
+   * In case of decryption, data can be an Array of objects or an object
+   * @param {string} action - action to perform. Can be either ENCRYPT or DECRYPT
```

Comment:
Sorry I should have been more clearer. Sine you already have Action just , in jsdoc, declare it as a type (or a class or enum) and then here you can set that type instead of string (this is nice for IDEs that interpret this.)
http://usejsdoc.org/tags-enum.html